app.js

```diff
@@ -0,0 +1,204 @@
+
+
+// Q1
+let nickname = 'ごっしー';
+let age = '28';
+let greet = '私のニックネームは' + nickname + 'です。年齢は'  + age + 'です。';
+console.log(greet);
+
+
+// Q2
+let languages = ['JavaScript', 'PHP', 'RUdy', 'Python','Go'];
```

Comment:
[must]
typoは気をつけましょう！
PR作成した際のFile Changedで必ず確認するようにしましょう！

```suggestion
let languages = ['JavaScript', 'PHP', 'Ruby', 'Python', 'Go'];
```

Reply:
承知しました。
修正いたしました。